core/src/test/java/apoc/schema/SchemasEnterpriseFeaturesTest.java

```diff
@@ -39,23 +34,15 @@ public class SchemasEnterpriseFeaturesTest {
 
     @BeforeClass
     public static void beforeAll() {
-        assumeFalse(isRunningInCI());
```

Comment:
Can you please just remove only this line?
We want to skip Container tests in case the environment does not have Docker up and running

Reply:
Hi Andrea, it's my first PR in this repo so apologies if I didn't know how it is supposed to work. 

I saw this line in the file but I didn't really understand why we would not want these tests to run in the CI. What is the reasoning behind that? 

It looks like the Github job has passed and the TeamCity job should pass too (not sure when it's run though). One thing which you may or may not be aware of is that recently Nacho has added docker to TeamCity/Github I believe.

Reply:
Hi Andrea, 

Sorry to bother you with this but if I would like to see if I can merge this today. I'm thinking that if I don't hear back from you before tomorrow then I will assume that you are OK with this PR and that it's fine for these tests to run in CI 🙇